launcher/ui/WinDarkmode.h

```diff
@@ -0,0 +1,70 @@
+#ifndef WINDARKMODE_H
```

Comment:
we generally use `#pragma once` instead of those guards, so it would be interesting to change it to be uniform with the rest of the code base :)